packages/core/src/domain/configuration/configuration.ts

```diff
@@ -107,6 +108,7 @@ export function buildConfiguration(initConfiguration: InitConfiguration, buildEn
       initConfiguration.beforeSend && catchUserErrors(initConfiguration.beforeSend, 'beforeSend threw an error:'),
     cookieOptions: buildCookieOptions(initConfiguration),
     service: initConfiguration.service,
+    sdkVersion: buildEnv.sdkVersion,
```

Comment:
FMU, we prefer arguments instead of static import, like in "buildConfiguration". 
Instead of adding `buildEnv` all the way down, my approach is to merge it to the configuration. So we send it to `buildConfiguration()` and we get the configuration with everything we need for the rest of the code.
Wdyt?

Reply:
FMU, buildEnv is a constant, we import constant in a lot of places.